tests/source/expr.rs

```diff
@@ -471,14 +471,18 @@ fn issue3226() {
     {
         {
             {
-                return Err(ErrorKind::ManagementInterfaceError("Server exited unexpectedly").into())
+                {
+                    return Err(ErrorKind::ManagementInterfaceError("Server exited unexpectedly").into())
+                }
             }
         }
     }
     {
         {
             {
-                break Err(ErrorKind::ManagementInterfaceError("Server exited unexpectedlyy").into())
+                {
+                    break Err(ErrorKind::ManagementInterfaceError("Server exited unexpectedly").into())
+                }
```

Comment:
Some of these things seemed to have been very particular about line length. I've inserted an extra layer of `{`/`}` pairs to try to honor that while performing the correction:

```diff
-unexpectedlyy
+unexpectedly
```